src/services/DavRequest.js

```diff
@@ -20,17 +20,19 @@
  *
  */
 const props = `
-	<oc:fileid />
-	<d:getlastmodified />
-	<d:getetag />
-	<d:getcontenttype />
 	<d:getcontentlength />
-	<nc:realpath />
-	<nc:has-preview />
-	<nc:file-metadata-size />
+	<d:getcontenttype />
+	<d:getetag />
+	<d:getlastmodified />
+	<d:resourcetype />
 	<nc:face-detections />
+	<nc:file-metadata-size />
+	<nc:has-preview />
+	<nc:realpath />
 	<oc:favorite />
-	<d:resourcetype />`
+	<oc:fileid />
+	<oc:permissions />
+`
```

Comment:
Any reason for the reordering ? just curious :)

Reply:
just to mess with us :D

Reply:
Sorting alphabetically